test/unittests/tracing_test.cpp

```diff
@@ -95,3 +95,16 @@ TEST_F(tracing, three_tracers)
 
     EXPECT_EQ(trace(dup1(0)), "A0:PUSH1 B0:PUSH1 C0:PUSH1 A2:DUP1 B2:DUP1 C2:DUP1 ");
 }
+
+TEST_F(tracing, histogram)
+{
+    vm.add_tracer(evmone::create_histogram_tracer(trace_stream));
+
+    trace_stream << '\n';
+    EXPECT_EQ(trace(add(0, 0)), R"(
+--- # HISTOGRAM depth=0
```

Comment:
Is this valid in CSV?

Reply:
No. But this is yaml document start mark.

Reply:
The CSV can only have optional header line with column names. You cannot have any title line.

The fancier YAML wrapper would be
```yaml
--- # HISTOGRAM
depth: 0
csv: |
  opcode,count
  ADD,1
  PUSH1,2
```

Reply:
So why not output YAML and then whoever wants CSV can do the conversion themselves (or in worst-case by skipping the top 3 lines) ?

Reply:
Because nobody wants YAML. I used `---` because of no better ideas.